### DISCUSSION TITLE
二次封装的在线工具

### DISCUSSION THREAD

**Discussion by geek-bigniu** [echo094/decode-js#59] [Show and tell]
地址：https://blog.168api.cn/jsjiami-com-v7-%e5%9c%a8%e7%ba%bf%e8%a7%a3%e5%af%86

**Comment by echo094** [comment]
建议你不要单独指出是jsjiami-com-v7，最好把其它方式也封装进去。

给你看个例子：NXY666/JsjiamiV6-Decryptor#1

**Comment by geek-bigniu** [comment]
> 建议你不要单独指出是jsjiami-com-v7，最好把其它方式也封装进去。
> 
> 给你看个例子：[NXY666/JsjiamiV6-Decryptor#1](https://github.com/NXY666/JsjiamiV6-Decryptor/discussions/1)

嗯嗯，感谢提醒
